bundles/org.openhab.binding.withings/src/main/resources/OH-INF/thing/person.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="withings"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="Person">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="withingsapi"/>
+		</supported-bridge-type-refs>
+
+		<label>Withings Person</label>
+		<description>Withings Person</description>
+
+		<channels>
+			<channel id="personWeight" typeId="weight"/>
+			<channel id="personHeight" typeId="height"/>
+			<channel id="personFatMass" typeId="fatMass"/>
+			<channel id="personLastSleepStart" typeId="lastSleepStart"/>
+			<channel id="personLastSleepEnd" typeId="lastSleepEnd"/>
+			<channel id="personLastSleepScore" typeId="lastSleepScore"/>
+		</channels>
+
+		<properties>
+			<property name="userId"/>
+		</properties>
+
+		<representation-property>id</representation-property>
+		<config-description-ref uri="thing-type:withings:person"/>
+	</thing-type>
+
+	<channel-type id="weight">
+		<item-type>Number</item-type>
```

Comment:
You could use Units of Measure/QuantityType.
```suggestion
		<item-type>Number:Mass</item-type>
```

Reply:
solved